apps/evm/lib/evm/gas.ex

```diff
@@ -347,23 +347,27 @@ defmodule EVM.Gas do
   end
 
   def operation_cost(:sstore, [key, new_value], _machine_state, exec_env) do
-    case ExecEnv.get_storage(exec_env, key) do
-      :account_not_found ->
-        @g_sset
-
-      :key_not_found ->
-        if new_value != 0 do
+    if Configuration.eip1283_sstore_gas_cost_changed?(exec_env.config) do
+      eip1283_sstore_gas_cost([key, new_value], exec_env)
+    else
+      case ExecEnv.get_storage(exec_env, key) do
```

Comment:
What do you think about extracting all this logic into a private function? That way you're dealing with the same level of abstraction in the `operation_cost` function. For example,

```elixir
def operation_cost(:sstore, [key, new_value], _machine_state, exec_env) do
  if Configuration.eip1283_sstore_gas_cost_changed?(exec_env.config) do 
    eip1283_sstore_gas_cost([key, new_value], exec_env)
else
  # you might be able to think of a better name for this function
   basic_sstore_gas_cost([key, new_value], exec_env)
end
```